README.rst

```diff
@@ -1874,823 +67,14 @@ as in the example below:
-            }
-        }
-    }
-    '''
-
-    # Compile query. Note that the edge traversal gets compiled to a SQL join.
-    compilation_result = graphql_to_sql(sql_schema_info, graphql_query, {})
-
-
-Including tables without explicitly enforced primary keys
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The compiler requires that each SQLAlchemy :code:`Table` object in the :code:`SQLALchemySchemaInfo`
-has a primary key. However, the primary key in the :code:`Table` need not be the primary key in
-the underlying table. It may simply be a non-null and unique identifier of each row. To override
-the primary key of SQLAlchemy :code:`Table` objects reflected from a database please follow the
-instructions in `this link
-<https://docs.sqlalchemy.org/en/13/core/reflection.html#overriding-reflected-columns>`__.
-
-Including tables from multiple schemas
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-SQLAlchemy and SQL database management systems support the concept of multiple `schemas
-<https://docs.sqlalchemy.org/en/13/core/metadata.html?highlight=schema#specifying-the-schema-name>`__.
-One can include :code:`Table` objects from multiple schemas in the same
-:code:`SQLAlchemySchemaInfo`. However, when doing so, one cannot simply use table names as
-GraphQL object names because two tables in different schemas can have the
-same the name. A solution that is not quite guaranteed to work, but will likely work in practice
-is to prepend the schema name as follows:
-
-.. code:: python
-
-    vertex_name_to_table = {}
-    for table in metadata.values():
-        # The schema field may be None if the database name is specified in the connection string
-        # and the table is in the default schema, (e.g. 'dbo' for mssql and 'public' for postgres).
-        if table.schema:
-            vertex_name = 'dbo' + table.name
-        else:
-            # If the database name is not specified in the connection string, then
-            # the schema field is of the form <databaseName>.<schemaName>.
-            # Since dots are not allowed in GraphQL type names we must remove them here.
-            vertex_name = table.schema.replace('.', '') + table.name
-
-        if vertex_name in vertex_name_to_table:
-            raise AssertionError('Found two tables with conflicting GraphQL object names.')
-
-        vertex_name_to_table[vertex_name] = table
-
-Including manually defined :code:`Table` objects
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The :code:`Table` objects in the :code:`SQLAlchemySchemaInfo` do not need to be reflected from the
-database. They also can be manually specified as in `this link
-<https://docs.sqlalchemy.org/en/13/core/metadata.html#creating-and-dropping-database-tables>`__.
-However, if specifying :code:`Table` objects manually, please make sure to include a primary key
-for each table and to use only SQL types allowed for the dialect specified in the
-:code:`SQLAlchemySchemaInfo`.
-
-Macro system
-------------
```

Comment:
I verified that the macro system section is essentially copy pasted in ReadTheDocs.